src/Sulu/Bundle/SecurityBundle/Build/UserBuilder.php

```diff
@@ -33,7 +34,7 @@ public function build()
         $user = 'admin';
         $password = 'admin';
         $roleName = 'User';
-        $system = 'Sulu';
+        $system = Admin::SULU_ADMIN_SECURITY_SYSTEM;
```

Comment:
That's where it gets weird IMO, also a reason I actually would prefer not to have this constant.

Reply:
why do you feel this is weird? the builder creates the default user for the admin, right?

Reply:
To me it feels more like the security system itself should have control over how this is called, but it only makes it a little bit better... And that I guess this is only a temporary solution until we have a proper fix (which would even include getting rid of the array being returned from the `getSecurityContexts` function) it will continue to feel weird to me... That's why I would not introduce this variable at this point. But as said, if you insist on it I can live with it.

Reply:
alright. properly fixing this would be nice indeed - not sure if it is a good idea that a security system is just a string value in the first place.. but until then, i would prefer to use a constant instead of continue using tha string without context 😇